src/atomate2/forcefields/flows/phonons.py

```diff
@@ -131,191 +117,31 @@ class PhononMaker(Maker):
     prefer_90_degrees: bool = True
     get_supercell_size_kwargs: dict = field(default_factory=dict)
     use_symmetrized_structure: str | None = None
-    bulk_relax_maker: BaseVaspMaker | ForceFieldRelaxMaker | None = field(
+    create_thermal_displacements: bool = True
+    generate_frequencies_eigenvectors_kwargs: dict = field(default_factory=dict)
+    kpath_scheme: str = "seekpath"
+    store_force_constants: bool = True
+
+    code: str = "forcefields"
+    bulk_relax_maker: ForceFieldRelaxMaker | None = field(
         default_factory=lambda: CHGNetRelaxMaker(relax_kwargs={"fmax": 0.00001})
```

Comment:
```suggestion
        default_factory=lambda: CHGNetRelaxMaker(relax_cell=True, relax_kwargs={"fmax": 0.00001})
```

While looking at the code in a different context again: could we add a volume relaxation here. Sorry, probably something I had overlooked.